homeassistant/components/derivative/sensor.py

```diff
@@ -93,6 +98,10 @@ def __init__(
 
         self._unit_prefix = UNIT_PREFIXES[unit_prefix]
         self._unit_time = UNIT_TIME[unit_time]
+        if time_window is None:
+            self._time_window = 0
```

Comment:
Instead of specifying a default value here, we should declare a `DEFAULT_TIME_WINDOW` in the config schema and set it to `0`.

Reply:
Done!